monitor/services/src/main/java/org/datacleaner/monitor/server/components/ComponentHandler.java

```diff
@@ -74,43 +76,48 @@
 
     public static ObjectMapper mapper = Serializator.getJacksonObjectMapper();
 
-    private final String componentName;
-    private DataCleanerConfiguration dcConfiguration;
+    private final String _componentName;
+    private final DataCleanerConfiguration _dcConfiguration;
+    private final StringConverter _stringConverter;
```

Comment:
You're right. Hmm maybe we should internally in StringConverter create a kind of a "base converter delegate" and then only wrap it with further converters when needed (basically when the collection in ```StringConverter.deserialize(String, Class<E> type, Collection<Class<? extends Converter<?>>>)``` is not empty).